.github/workflows/CI.yml

```diff
@@ -33,3 +33,5 @@ jobs:
       run: cargo rustdoc --lib --all-features --examples
     - name: Run tests
       run: cargo test --verbose --all-features
+    - name: Run timezone-sensitive tests in non UTC (GMT-5)
+      run: TZ="std +5" cargo test --verbose --all-features graph
```

Comment:
So this test runs twice, once "normally" and once with a custom TZ?

Reply:
Correct. I think that CI runs them in UTC since they passed when I am pretty sure they should fail with non UTC timezone. So I added this. It is a very short and quick test.